client/src/App.jsx

```diff
@@ -51,6 +52,14 @@ const router = createBrowserRouter(
           </Protected>
         }
       />
+      <Route
+        path="questions/:id"
```

Comment:
```suggestion
        path="questions/:questionId"
```

Reply:
The convention used by most frameworks is `id`. Look at the [Rails Routing](https://guides.rubyonrails.org/routing.html) doc for example. Idea is that it correlates to the primary key